tests/wpunit/PostObjectFieldsTest.php

```diff
@@ -54,6 +54,7 @@ public function register_acf_field_group( $config = [] ) {
 			'description'           => '',
 			'show_in_graphql'       => 1,
 			'graphql_field_name'    => 'postFields',
+			'graphql_types_on'		=> ['post_type__post']
```

Comment:
Even with these changes, tests are still failing 🤔

Reply:
There was a typo and it caused a warning with `user_register_field_groups`. Fixed the warning with new commit.

Reply:
Ok, checking it out 👀

Reply:
Tests are all still failing, but it's saying that it's because of `undefined function graphql()` so this might be a test environment issue. It seems like WPGraphQL isn't being loaded properly. 

I'll check it out and see if I can get that fixed.

Reply:
Ok, so I've confirmed that the tests run locally for me. And all but one pass now. 
So this seems like some sort of CI environment issue.